frontend/src/components/ProjectCard/index.jsx

```diff
@@ -7,46 +7,52 @@ import Image from 'next/image'
 import useThumbnail from '@hooks/useThumbnail'
 import styles from './index.module.scss'
 
-const ProjectCard = ({ projectName, summary, ownerName, repoName }) => {
-  const isSingleProject = projectName === '_'
-  const nameOnCard = isSingleProject ? repoName : projectName
-  const { src, isLoading, isError } = useThumbnail(
-    `${ownerName}/${repoName}`,
-    projectName,
-  )
-  return (
-    <Link
-      passHref
-      href={`/${ownerName}/${repoName}${isSingleProject ? '' : '/' + projectName}`}
-    >
-      <Card as="a" className={styles.card} data-cy="project-card">
-        <div className={styles.thumbnail}>
-          <div>
-            {isLoading || isError ? null : (
-              <Image
-                alt={`Render of PCB of ${nameOnCard}`}
-                data-cy="project-card-thumbnail"
-                height={180}
-                objectFit="contain"
-                src={src}
-                width={240}
-              />
-            )}
+const ProjectCard = forwardRef(
```

Comment:
Are you sure `forwardRef` is needed here? I think without it the `ref` just gets attached to `<Link>` automatically and this component doesn't need to worry about it.

Reply:
It's needed if you pass the ref as `<Component ref={value} />` I got the following error without `forwardRef`
![image](https://user-images.githubusercontent.com/37152329/204012281-895d1652-dd8a-4779-91dc-a969f1df9d56.png)

I got rid of `forwardRef` by passing the ref under a different name :point_down: 
```jsx
<ProjectCard
    intersectionObserverRef={
      index === projects.length - cardsPerRow * 2
        ? intersectionObserverRef
        : null
    }
 />
```